trace/trace.go

```diff
@@ -3,45 +3,17 @@ package trace
 import (
 	"context"
 	"github.com/signalfx/golib/log"
+	"github.com/signalfx/golib/trace/format"
 )
 
-// DefaultLogger is used by package structs that don't have a default logger set.
-var DefaultLogger = log.DefaultLogger.CreateChild()
+// Span is an alias for the real json span
+type Span traceformat.Span
```

Comment:
good point, i'm going to rethink this...

Reply:
So did some experimentation, if you're going to construct one of these, you will need to use the traceformat.Endpoint anyways so there's no benefit.  There's no way to ignore a particular file in the linters and code coverage just directories, so this is the best i can come up with. It's still a breaking change.  Thoughts?

Reply:
How about flipping it so that the main structs are in `./trace` and the type declarations that reference those are in `./trace/format` like so:

```
package traceformat

import "github.com/signalfx/golib/trace"

//easyjson:json
type Trace []*trace.Span

//easyjson:json
type Span trace.Span

//easyjson:json
type Endpoint trace.Endpoint

//easyjson:json
type Annotation trace.Annotation
```

I did a quick check with that and easyjson generated the parser for it fine and it at least compiled and passed the tests.  Not sure what other implications there are for doing that.

Reply:
that's a great suggestion. I'll give it a whirl and see how it pans out for importing in proxy etc. thanks.